api/src/pcapi/core/users/api.py

```diff
@@ -663,6 +665,17 @@ def update_user_info(
             snapshot.set("idPieceNumber", old=user.idPieceNumber, new=id_piece_number)
         user.idPieceNumber = id_piece_number
 
+    if user.pro_new_nav_state and new_nav_date is not UNCHANGED:
+        user.pro_new_nav_state.newNavDate = new_nav_date if new_nav_date is not None else None
```

Comment:
```suggestion
        user.pro_new_nav_state.newNavDate = new_nav_date
```
C'est pareil du coup, non ?